kubernetes/host-logs-geneva.yaml

```diff
@@ -56,10 +56,13 @@ spec:
        livenessProbe:
           exec:
             command:
-              - cmd
-              - /c
-              - .\opt\hostlogswindows\scripts\cmd\livenessprobe.exe
-              - ".\\etc\\hostlogswindows\\filesystemwatcher.txt"
+              - C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
+              - -command
+              - |
+                .\opt\hostlogswindows\scripts\powershell\livenessprobe.ps1 -exeRelativePath "\\opt\\genevamonitoringagent\\genevamonitoringagent\\Monitoring\\Agent\\MonAgentCore.exe" -fileName "etc\hostlogswindows\filesystemwatcher.txt"
```

Comment:
Should we watch MonAgentLauncher instead? MonAgentLauncher will spin up multiple instances of MonAgentCore and has built in health checks for these. In the case all instances of MonAgentCore crash, MonAgentLauncher should automatically spin up new instances

Reply:
Yes good point, if MonAgentLauncher isnt there nothing is running.

MonAgentLauncher triggers MonAgentHost

MonAgentHost makes sure MonAgentManger and MonAgentCore is running.

MonAgentManager just check resoruce usage of MonAgentCore.

https://eng.ms/docs/products/geneva/collect/references/agent/monagenthost